source/development/wazuh-logtest.rst

```diff
@@ -0,0 +1,63 @@
+.. Copyright (C) 2020 Wazuh, Inc.
+
+.. _dev-wazuh-logtest:
+
+
+Wazuh Logtest
+=============
+
+The Wazuh-Logtest solution is distributed by different components of the Wazuh environment:
+
+**Core Logtest**
+
+    The logtest core is part of the *Analysis daemon* (in manager). It has a session manager and shares its rules engine with Logtest.
+
+**Wazuh RESTful API**
+
+    Interact with Core Logtest, and provide an interface to test the rules and decoders remotely.
+
+**Kibana: Wazuh APP**
+
+    Wazuh application for Kibana uses RESTful API to support Logtest from the web interface.
+
+**Wazuh Logtest Tool**
+
+    Interact with Core Logtest to test the rules and decoders locally.
```

Comment:
Link to wazuh-logtest tool. Also referring it to be a replacement to ossec-logtest should be helpful. This also implicitly tell the user that resides at manager